### ISSUE TITLE
Box title is not displayed within some themes

### ISSUE THREAD

**Issue by kevinyuan** [plantuml/plantuml#1214]
Half of the built-in themes don't show the box title, e.g. cerulean.

![image](https://user-images.githubusercontent.com/231197/208011418-3489d2ab-5579-485b-ab3e-69b92b3078a3.png)

It looks like the themes with *-outline DO show the box title, but the horizontal divider lines cross the divider captions on the foreground. e.g. cerulean-outline

![image](https://user-images.githubusercontent.com/231197/208011480-4a31f5b2-e334-443d-978d-98f2a45f6446.png)

Please advice how to fix it.

Thank you very much.

**Comment by arnaudroques** [comment]
Could you post a minimum example that shows the issue?
This would help. Thanks!

**Comment by kevinyuan** [comment]
@startuml
!theme cerulean
 
box "S/W"
participant driver 

end box

box "NIC H/W"
participant dma_arb 
end box

== Receive Pkt ==
driver -> dma_arb: test

@enduml

**Comment by The-Lum** [comment]
Hello @kevinyuan, and all,

**Yes**: It seems that this part of code is now without effect:
https://github.com/plantuml/plantuml/blob/80217d8c900916ae8fa023478299ee0c9c78fbb2/themes/puml-theme-cerulean.puml#L291

A patch will be perhaps necessary (to transform all old `skinparam` to `style`) for all of them... :-)

Then a possible temporary workaround is to surcharge with adding some style as:

```puml
<style>
box {
  FontColor $DARK
}
</style>
```

And:

```puml
@startuml
!theme cerulean
<style>
box {
  FontColor $DARK
}
</style>

box "S/W"
participant driver

end box

box "NIC H/W"
participant dma_arb
end box

== Receive Pkt ==
driver -> dma_arb: test

@enduml
```

>[![](https://www.planttext.com/api/plantuml/svg/PS_12e9058NX_NsAaxGDz-68OKGHHDYYPOnw8McSaV4QHVJk3MMqQF-T7qxMY_Ooj8OcSkQMKR4V36jBIIztmocLxeO70Ijd9NV6UKoNYs9BJqhYaRnDT8YF4NMXrbHDfwsWzisLFH7R6a6CRBV9iVwZhJvfN_weKYYuuX30_Y9GYZurpD8ldaEu5w8iZCA15m00)](https://www.planttext.com/?text=PS_12e9058NX_NsAaxGDz-68OKGHHDYYPOnw8McSaV4QHVJk3MMqQF-T7qxMY_Ooj8OcSkQMKR4V36jBIIztmocLxeO70Ijd9NV6UKoNYs9BJqhYaRnDT8YF4NMXrbHDfwsWzisLFH7R6a6CRBV9iVwZhJvfN_weKYYuuX30_Y9GYZurpD8ldaEu5w8iZCA15m00)

_Enjoy,_
Regards.

**Comment by arnaudroques** [comment]
Thanks for the workaround.
This is now fixed [in last snapshot](https://github.com/plantuml/plantuml/releases/tag/snapshot) and [on the online server](https://www.plantuml.com/plantuml/uml/PSon2i8m58NX_Jx5iNjn5o855rr4T72KD3rWC4bBUYC-lW4LXExV_tUpsgmb1bdfdP5mp2NGfdOi6doYzECBpMLzRMIgqZi_sQGOidyoGvW6L24VTZpii5_GQ6yszt_M69pfM0yuFHJ6o7VNRdzw0-MiqjMcnF06).

Tell us if you find other issues!

**Comment by The-Lum** [comment]
_[Just for traceability to the author of this theme]_
FYI @bschwarz:

Here is a minor update on `uml-theme-cerulean-outline.puml`:
https://github.com/plantuml/plantuml/blob/8ff88645d4c3af088b551df84e93cd3f4e8851bc/themes/puml-theme-cerulean-outline.puml#L297

See PlantUML commit: https://github.com/plantuml/plantuml/commit/8ff88645d4c3af088b551df84e93cd3f4e8851bc#diff-e87e07ca1f415f3ccab6473c1b74f639eaed4ac8b941ce4a7bf35dfe07748370

Thanks PlantUML team, and @arnaudroques,
Regards.

**Comment by arnaudroques** [comment]
This should be fixed in V1.2023.0

**Comment by The-Lum** [comment]
Hi all,

_[This is an Issue Review]_
This is now fixed.
- Thanks @kevinyuan for the report.
- Thanks PlantUML team for your work.
	
Regards.
